data/sql_updates/large_aggregates.sql

```diff
@@ -123,7 +123,43 @@ party_totals as (
     group by
         month,
         year
-), -- merge
+),
+communication_totals as (
+  select
+    get_cycle(rpt_yr) as cycle,
+    f.rpt_yr as year,
+    extract(month from to_date(cast(cvg_end_dt as text), 'YYYY-MM-DD')) as month,
+    sum(coalesce(ttl_communication_cost ,0)) as comms_totals
+  from public.fec_vsum_f7_vw f
```

Comment:
I think this should be from `fec_fitem_f76_vw` Correct me if I am wrong, but the vsum tables had issues so we want to use the fitem tables